docs/topics/exclude-files-and-packages.adoc

```diff
@@ -16,7 +16,7 @@ Any packages that match the defined value are excluded. For example, you can use
 You can exclude packages by either of the following methods:
 
 * Using the `--excludePackages` argument.
-* Specifying the packages in a file contained within one of the ignored locations. Each package should be included on a separate line, and the file must end in `.package-ignore.txt`. For example, see `<MTA_HOME>/ignore/proprietary.package-ignore.txt`.
+* Specifying the packages in a file contained within one of the ignored locations. Each package should be included on a separate line, and the file must end in `.package-ignore.txt`. For example, see `<{ProductShortName}_HOME>/ignore/proprietary.package-ignore.txt`.
```

Comment:
Did QE check this?

Reply:
Then type  in "mtr" and have QE check. (general rule for file and directory names).

Reply:
@RichardHoch Actually, the preview does render the attributes. So it was checked.